core/modules/cart/store/actions/totalsActions.ts

```diff
@@ -10,6 +10,9 @@ import {
 import EventBus from '@vue-storefront/core/compatibility/plugins/event-bus'
 
 const totalsActions = {
+  async syncTotalsWhenLocalStorageChange ({ commit }, payload) {
```

Comment:
wdyt about just `updateTotals` or something similar? I understand the context where it's used, but it just updates totals 🤷I think it doesn't need to have such a specific name

Reply:
Done. I did same for updateCart